src/game/logic/object/object.cpp

```diff
@@ -66,3 +69,102 @@ int Object::Get_Indicator_Color() const
     return 0;
 #endif
 }
+
+bool Object::Is_Locally_Controlled() const
+{
+    return g_thePlayerList->Get_Local_Player() == Get_Controlling_Player();
+}
+
+void Object::Get_Health_Box_Position(Coord3D &pos) const
+{
+    pos = *Get_Position();
+    pos.z += Get_Geometry_Info().Get_Max_Height_Above_Position() + 10.0f;
+    pos.Add(&m_healthBoxOffset);
+
+    if (Is_KindOf(KINDOF_MOB_NEXUS)) {
+        pos.z += 20.0f;
+    }
+}
+
+bool Object::Get_Health_Box_Dimensions(float &width, float &height) const
+{
+    if (Is_KindOf(KINDOF_IGNORED_IN_GUI)) {
+        width = 0.0f;
+        height = 0.0f;
+        return false;
+    } else {
+        float f1;
+
+        if (Get_Geometry_Info().Get_Minor_Radius() + Get_Geometry_Info().Get_Major_Radius() <= 150.0f) {
+            f1 = Get_Geometry_Info().Get_Minor_Radius() + Get_Geometry_Info().Get_Major_Radius();
+        } else {
+            f1 = 150.0f;
+        }
+
+        float f2;
```

Comment:
Given the f2 code below, it can be simplified to one line, because f2 is computed the same as f1 here.

```c++
float f2 = std::max(f1, 20.0f);
```

Reply:
fixed.